README.md

```diff
@@ -172,6 +172,25 @@ For example:
 ┣ ┣ ┃ ┃ ┣ 📜 UserProfile.stories.mdx \
 ```
 
+#### Scoped Files 
+
+We need to add inside `pages/**/{utils, helpers, context, hooks, etc...}` and use `camelCase` as **Naming Convention**.
+
+If we need to used this files again in another context or page it need to be moved to `src/{utils, helpers, context, hooks}`.
+
+For example:
+
+```sh
+┣ 📂 pages \
+┣ ┣ 📂 Home \
+┣ ┃ ┣ 📂 utils \
+┣ ┃ ┣ ┣ 📜 someUtils.js
+┣ ┃ ┣ 📂 helpers \
+┣ ┣ ┣ ┣ 📜 someHelper.js
+┣ ┃ ┣ 📂 hooks \
+┣ ┣ ┣ ┣ 📜 useSomeHook.js
+```
+
```

Comment:
Suggestion: This example is for the first scenario, maybe we could put it right after the description of it. Markdown got messed up, but I'll try to show what I'm thinking:

```
We need to add inside `pages/**/{utils, helpers, context, hooks, etc...}` and use `camelCase` as **Naming Convention**.

[your exemple here]

If we need to used this files again in another context or page it need to be moved to `src/{utils, helpers, context, hooks}`.
```

Reply:
Nice suggestion!! I created the examples for when moving these guys to a global context :)
thanks, done in [f642d5f](https://github.com/juntossomosmais/frontend-guideline/pull/29/commits/f642d5f942236ce61c1602bd8b1f76ecaa25d364)